http-client/src/test/java/org/triplea/http/client/error/reporting/WireMockSystemTest.java.orig

```diff
@@ -25,6 +25,10 @@
 import org.apache.http.HttpStatus;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+<<<<<<< HEAD:http-client/src/test/java/org/triplea/http/client/WireMockSystemTest.java
+=======
+import org.triplea.http.client.ServiceCallResult;
+>>>>>>> Group error reporting http client classes together in a package:http-client/src/test/java/org/triplea/http/client/error/reporting/WireMockSystemTest.java
```

Comment:
Errm I'm pretty sure that's a bad merge conflict resolution

Reply:
File removed: 7590176ab